### PR TITLE
Update spinbutton.rb

### PR DESCRIPTION
**Pull request by arjunmenon**
Updated to new code format.

### CODE REVIEW

File: gtk3/sample/gtk-demo/spinbutton.rb [hunk 1]
```diff
@@ -4,15 +4,15 @@
 =begin
 =  Spin Button
 
- GtkSpinButton provides convenient ways to input data
+ GtkSpinButton2 provides convenient ways to input data
```
Comment (kou) [inline gtk3/sample/gtk-demo/spinbutton.rb]: Is this change needed?

File: gtk3/sample/gtk-demo/spinbutton.rb [hunk 2]
```diff
@@ -104,20 +109,23 @@ def self.time_spin_input(button)
...
 
-  def self.month_spin_output(button)
+  def month_spin_output(button)
     value = button.adjustment.value || 1
     button.text = Date::MONTHNAMES[value]
     true
   end
+
+  
+
```
Comment (kou) [inline gtk3/sample/gtk-demo/spinbutton.rb]: Please don't add needless spaces.

Comment (arjunmenon) [inline gtk3/sample/gtk-demo/spinbutton.rb]: Updated.

**Comment by kou** [comment]
Thanks!!!!!
